source/adios2/engine/bp4/BP4Reader.cpp

```diff
@@ -495,6 +495,13 @@ void BP4Reader::InitBuffer(const TimePoint &timeoutInstant, const Seconds &pollS
                         "the writer is creating the new files.");
             }
         }
```

Comment:
This looks unrelated to the other changes.

Reply:
Removed.